packages/design-system/src/styles/settings/variables/_shadow.scss

```diff
@@ -8,3 +8,8 @@ $shadow-base-blur-radius: 4px;
 $shadow-base-color: rgba(0, 0, 0, 0.25);
 $shadow-base: $shadow-base-offset-x $shadow-base-offset-y $shadow-base-blur-radius
   $shadow-base-color;
+
+// Inherited from USWDS
+$uswds-box-shadow: 0 0 2px $shadow-base-color !default;
+$uswds-focus-shadow: 0 0 3px $color-focus, 0 0 7px $color-focus !default;
```

Comment:
I would also prefer to remove these. The few places that use these variables should just use the hardcoded value, and ideally using the `$focus-color` variable